CMakeLists.txt

```diff
@@ -97,6 +104,9 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 # Code coverage is optional and OFF by default
 option(CODECOVERAGE "Enable code coverage for the build" OFF)
 
+# Generate downgrade script
+option(GENERATE_DOWNGRADE_SCRIPT "Generate downgrade script" ON)
```

Comment:
hmm should the default be OFF

Reply:
Maybe not, but it's very hard to tell from the README.md....

Reply:
Changed it to `OFF`.